examples/node-browser/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "hawkular-apm-opentracing-node-browser-example",
+  "version": "0.0.1",
+  "description": "Hawkular APM OpenTracing example with Node.js and browser",
+  "license": "Apache-2.0",
+  "author": "Pavol Loffay <ploffay@redhat.com>",
+  "main": "index.js",
+  "scripts": {
+    "webpack": "npm --prefix ../../ install && webpack --display-error-details",
```

Comment:
I think this looks OK, but to be honest, I don't have a lot of experience with web pack